interpreter/cling/tools/driver/CMakeLists.txt

```
@@ -9,9 +9,9 @@
 # Keep symbols for JIT resolution
 set(LLVM_NO_DEAD_STRIP 1)
 
+set(LLVM_LINK_COMPONENTS support)
```

Should be
```suggestion
set(LLVM_LINK_COMPONENTS Support)
```
I think (looking at upstream LLVM sources)

---

I think they are normalized. But capitalized like this, it's more llvmish, :+1: